### DISCUSSION TITLE
What number is associated with the varieties of Heinz?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#978] [Galaxy Trivia] [answered]
**Bonus:** What was their first product?

**Comment by peterbaccaro** [comment, marked as answer]
The number **57** is associated with the varieties of Heinz.

Heinz 57 is a synecdoche of the historical advertising slogan "57 Varieties" by the H. J. Heinz Company located in Pittsburgh, Pennsylvania, United States. It was developed from the marketing campaign that told consumers about the numerous products available from the Heinz company.

The very first Heinz product was **bottled horseradish** which saved homemakers from a thankless but necessary task. The preparation of horseradish as a condiment required long and tedious grating.
